README.md

```diff
@@ -87,11 +89,27 @@ import { AppComponent } from './app.component';
     BrowserModule
   ],
   providers: [],
-  schemas: [CUSTOM_ELEMENTS_SCHEMA],
-  bootstrap: [AppComponent]
+  schemas: [ CUSTOM_ELEMENTS_SCHEMA ],
+  bootstrap: [ AppComponent ]
 })
 export class AppModule { }
 
+// or the vanilla JS version
+
+(function(app) {
+  app.AppModule =
+    ng.core.NgModule({
+      imports: [ ng.platformBrowser.BrowserModule ],
+      declarations: [ app.AppComponent ],
+      schemas: [ ng.core.CUSTOM_ELEMENTS_SCHEMA ],
+      bootstrap: [ app.AppComponent ]
+    })
+    .Class({
+      constructor: function() {}
+    });
+})(window.app || (window.app = {}));
+
+
 /*
 You can now use the custom element in either a standalone or inline template
```

Comment:
It does need nanocompoent. You just use the nanocomponent to custom element adapter and that's it. Once you've registered your nanocomponent as a custom element and added to CUSTOM_ELEMENTS schema to your angular module, you're done and you can use the nanocomponent as a custom element inside any angular template.

It looks something like this:
https://snag.gy/NJxhLy.jpg